benchmarks/expression_problem/oop_c#/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace oop_c_
+{
+    class Program
+    {
+        static Random rand = new Random(2);
+        static int number = 0;
+
+        static void Main(string[] args)
+        {
+            int printCount = 0;
+            int evalCount = 0;
+            for (int i = 0; i < 1000; i++)
+            {
+                number = 0;
+                IExpression expr = generateRandomExpression(1000);
+                printCount += expr.PrettyPrint().Length;
+                evalCount += expr.Eval();
+            }
+            System.Console.WriteLine(printCount);
+            System.Console.WriteLine(evalCount);
+        }
+
+        static IExpression generateRandomExpression(int max)
+        {
+            if(number >= max) return new Lit(rand.Next(0, 100));
+            else
+            {
+                number++;
+                int exprChoice = rand.Next(0, 4);
+                if(exprChoice == 0) return new Add(generateRandomExpression(max), generateRandomExpression(max));

```

Comment:
Tænker det kunne se godt ud med en switch case :)

Reply:
I done it